Application/src/Main.java

```diff
@@ -8,6 +8,11 @@
 import javafx.scene.control.TabPane;
 import javafx.scene.layout.BorderPane;
 import javafx.scene.layout.HBox;
+import javafx.scene.layout.Background;
```

Comment:
Could be squishes to import *.

Reply:
Yeah, I know, I just didn't want to import something too big; keep it clean and whatnot